app/assets/stylesheets/layout_components/forms.scss

```diff
@@ -63,6 +63,84 @@ form {
   }
 }
 
+/* ------------- Switch (Toggle Button) Styles ------------- */
+.switch {
+  position: relative;
+  display: inline-block;
+  width: 64px;
+  height: 30px;
+  float: right;
+
+  &-slider {
```

Comment:
there are no single dashes used in BEM. If you need to have these different classes between the switch and switch-slider then just create two separate classes that are not nested and apply both of those to the html.